### DISCUSSION TITLE
Cast send" command not reading RPC URL from .env file (Foundry)

### DISCUSSION THREAD

**Discussion by chapman91** [Cyfrin/foundry-full-course-cu#1933] [Q&A]
I'm encountering an error message "a value is required for '--rpc-url <URL>' but none was supplied" when using the cast send command in Foundry. While I have the RPC URL stored in a .env file, the command doesn't seem to be reading it automatically.

How to configure cast to read environment variables from a .env file?

**Comment by EngrPips** [comment]
Hello @chapman91, did you run `source .env` before running the `cast send` command?

**Reply by chapman91** [reply]
No, I did not 

I got it to work.

Thank you!

**Reply by EngrPips** [reply]
Awesome. You are a Genius.
